### PR TITLE
Add new Helm chart and Dockerfile for imagepullsecret-patcher

### PR DESCRIPTION
**Pull request by pj-linebaugh**
Adding chart used for getting around the image download rate-limiting of the Docker repository.

None of you have probably even noticed this deployment on the clusters, but it has been great to get around the image download rate-limiting of Docker that has been going on since November 2020.  Trying out the gitflow process.

### CODE REVIEW

File: imagepullsecret-patcher/.gitignore [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
+.vscode/
+.DS_Store
```
Comment (cschreep) [inline imagepullsecret-patcher/.gitignore:18]: This file seems unnecessary if this isn't a separate git module

File: imagepullsecret-patcher/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# build stage
```
Comment (cschreep) [inline imagepullsecret-patcher/Dockerfile:1]: It seems like the whole application repo has been put in the helm chart. That seems like a counterintuitive location. Is there any reason not to have our own repo or fork if we need to build the docker image ourselves?

**Review by cschreep — CHANGES_REQUESTED** [review]

File: imagepullsecret-patcher/.gitignore (hunk 1 above)
Comment (pj-linebaugh) [reply imagepullsecret-patcher/.gitignore:18]: yep, i'll get rid of it.

File: imagepullsecret-patcher/Dockerfile (hunk 2 above)
Comment (pj-linebaugh) [reply imagepullsecret-patcher/Dockerfile:1]: Was wondering about that.  Maybe now is the time to figure out the best way to have the helm chart code live within the same repo as the app and reference it from the devops repo like we do for loki/kubecost.

Comment (cschreep) [reply imagepullsecret-patcher/Dockerfile:1]: Good call. We should have a meeting to think about our options.

**Review by n0mn0m — COMMENTED** [review]
Echo the thoughts on it where the actual application code should live, but otherwise looks good.

File: imagepullsecret-patcher/Dockerfile (hunk 2 above)
Comment (pj-linebaugh) [reply imagepullsecret-patcher/Dockerfile:1]: I'll do that and drop this PR.

**Pull request closed without merging**